styles.css

```diff
@@ -0,0 +1,413 @@
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #f9f9f9;
+  display: block;
+  padding: 0;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: rgb(22, 22, 22);
+  font-weight: normal;
+  font-style: normal;
+}
+
+.upload-signin {
+  margin-right: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav {
+  position: sticky;
+  top: 0;
+  width: 100%;
```

Comment:
By default, elements with a display of `block` (like the `nav` element) take 100% of the width of their parent elements.